src/test/java/com/notably/logic/suggestion/SuggestionTestUtil.java

```diff
@@ -0,0 +1,109 @@
+package com.notably.logic.suggestion;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.notably.commons.path.AbsolutePath;
+import com.notably.model.Model;
+import com.notably.model.ModelManager;
+import com.notably.model.block.BlockModel;
+import com.notably.model.suggestion.SuggestionItem;
+import com.notably.model.suggestion.SuggestionItemImpl;
+import com.notably.model.suggestion.SuggestionModel;
+import com.notably.model.suggestion.SuggestionModelImpl;
+import com.notably.model.viewstate.ViewStateModel;
+import com.notably.model.viewstate.ViewStateModelImpl;
+import com.notably.testutil.TypicalBlockModel;
+
+/**
+ * Contains helper methods for testing suggestion feature.
+ */
+public class SuggestionTestUtil {
+    private static Model model;
```

Comment:
I don't really like having a static model here, just like I mentioned yesterday.

> Hmm tbh, I don't really like these static variables over here. Using static fields means that you'd only have one instance of toCs2103 (for example) per JVM, which I don't think is right since we would like to isolate each test class as much as possible (even though JUnit actually runs test cases sequentially, so this would still perform as expected).

I'm pretty sure this can be restructured in a way that does not require this field!